lib/service/contact_service.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:convert';
+import 'dart:io';
+import '../model/contact.dart';
+import '../model/contact_data.dart';
+import '../utils/constants.dart';
+import 'interface/i_contact_service.dart';
+import 'package:http/http.dart' as http;
+
+class ContactService extends IContactService {
+  http.Client client = http.Client();
```

Comment:
This is not necessary, instead to define the type of the class, you can replace it with var